packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -66,54 +77,3 @@ export async function activate(context: ExtensionContext) {
   // client can be deactivated on extension deactivation
   context.subscriptions.push(disposable);
 }
-
-function isLWCProject(): boolean {
```

Comment:
These parts have been moved to the lwc-language-server so there is no need to duplicate the logic here.